### ISSUE TITLE
Delay before keyboard capture is active?

### ISSUE THREAD

**Issue by BenoitDuffez** [alainfrisch/nostalgy#112]
Running TB 102.13.0 on Ubuntu 23.04

Sometimes when I'm too fast, I press `sxxxx` to move the email to folder `xxxx` but instead it doesn't send my key presses to Nostalgy++ but rather it handles this through Thunderbird. The `s` will star the message, and whatever letters the folder is made up of (`xxxx`) will trigger a range of unexpected and random actions (compose email, forward, archive, open this or that, etc).

It seems there is a delay before which Nostalgy++ is fully active.  
Is this configurable? Is there a way to avoid this?

**Comment by BenoitDuffez** [comment]
I have installed tbkeys-lite and added `"s":""` to the conf, so that `s` is no longer a Thunderbird shortcut. Disables the "star" shortcut, but it also disables Nostalgy++.

Perhaps there is a way to disable the built-in shortcut and keep Nostalgy++'s? I don't want to retrain close to a decade of muscle memory :)

**Comment by eliotmoss** [comment]
On 8/8/2023 12:16 PM, Benoit Duffez wrote:
> Well... might as well ignore this ticket, feel free to close as invalid/won't do.
> 
> I have installed tbkeys-lite and added |"s":""| to the conf, so that |s| is no longer a Thunderbird 
> shortcut. Works well for me!

I couldn't figure out what you did here that worked.
I installed tbkeys-lite and added a (main) binding
is "s": "".  What happened is that the s key does
nothing (unsurprisingly).  I do not get the prompt /
small windows from nostalgy for entering a folder name.

Best - Eliot Moss

**Comment by BenoitDuffez** [comment]
Yup sorry about that. I had tested that it disabled the star shortcut, while Nostalgy++ was set to `d`. After too many failed strokes on `s` (muscle memory) I reverted the Nostalgy++ config to `s` and noticed that didn't work... So I edited my post above. Sorry!

**Comment by BenoitDuffez** [comment]
Looks like we could call javascript from tbkeys-lite? I've stumbled upon this: https://github.com/wshanks/tbkeys/issues/94 and it looks like `NostalgyCmd('Move messages to:', NostalgyMoveToFolder, true);` would do the trick (it triggers the appropriate action when entered in the developer console, so I assume that if tbkeys-lite could call this function, it would trigger the appropriate Nostalgy++ action.

ref: https://github.com/nostalgy/nostalgy/blob/master/chrome/content/nostalgy_keys.js
